gdnative-derive/src/native_script.rs

```diff
@@ -36,12 +36,20 @@ pub(crate) fn derive_native_class(input: TokenStream) -> TokenStream {
                 None
             };
 
+            let update_config_warning_on_set_snippet = match config.update_config_warning_on_set {
+                true => quote!(_owner.update_configuration_warning();),
+                false => quote!(),
+            };
+
             let label = config.path.unwrap_or_else(|| format!("{}", ident));
             quote!({
                 builder.add_property(#label)
                     #with_default
                     .with_ref_getter(|this: &#name, _| &this.#ident)
-                    .with_setter(|this: &mut #name, _, v| this.#ident = v)
+                    .with_setter(|this: &mut #name, _owner: ::gdnative::TRef<Self::Base>, v| {
+                        #update_config_warning_on_set_snippet
+                        this.#ident = v
```

Comment:
Perhaps the function should be called after the value is set? Technically Godot may decide to update it immediately after `update_configuration_warning` is called in a future version.

Reply:
Good point. Will update.